docs/reference/content/driver-scala/builders/aggregation.md

```diff
@@ -129,9 +130,9 @@ expression and outputs to the next stage a document for each distinct grouping.
 expression on which to group, and zero or more 
 [accumulators]({{< docsref "reference/operator/aggregation/group/#accumulator-operator" >}}) which are evaluated for each 
 grouping.  To simplify the expression of accumulators, the driver includes an 
-[`Accumulators`]({{< apiref "mongo-scala-driver" "org/mongodb/scala/model/Aggregates$" >}}) class with static factory methods for each of the supported 
-accumulators. In the example below, it's assumed that the `sum` and `avg` methods of the `Accumulators` class have been statically 
-imported. 
+[`Accumulators`]({{< apiref "mongo-scala-driver" "org/mongodb/scala/model/Accumulators$" >}}) singleton object with factory methods 
+for each of the supported accumulators.
+In the example below, it's assumed that the `sum` and `avg` methods of the `Accumulators` class have been statically imported.
```

Comment:
- The link was not to the right program element.
- `org.mongodb.scala.model.Accumulators` is not a class but a singleton object.
- Changed line breaks such that lines comply with our `max_line_length = 140` limit.